comtypes/test/test_portabledevice.py

```diff
@@ -0,0 +1,146 @@
+ # so we don't crash when the generated files don't exist
+from __future__ import annotations
```

Comment:
This `__future__` is not used in other modules in `comtypes`.
So you should use string literals as annotations instead of this.